Comment.js

```diff
@@ -232,5 +234,7 @@ Comment.propTypes = {
   reported: PropTypes.bool,
   updatedAt: PropTypes.string,
   username: PropTypes.string,
-  usernameTapAction: PropTypes.func
+  usernameTapAction: PropTypes.func,
+  this.props.additionalStyles: PropTypes.object,
```

Comment:
pourquoi this.props ici alors que pas sur les autres ?

Reply:
oopsy